includes/watchers/class-algolia-post-changes-watcher.php

```diff
@@ -9,6 +9,11 @@ class Algolia_Post_Changes_Watcher implements Algolia_Changes_Watcher {
 	 */
 	private $index;
 
+	/**
+	 * @var Array
+	 */
+	private $postsDeleted = array();
```

Comment:
`$postsDeleted` (and `$this->postsDeleted`) should be snake_case per [WordPress standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions) like so: `$posts_deleted`